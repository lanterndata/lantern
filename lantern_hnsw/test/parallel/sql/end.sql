-- This file contains invariants to be checked after the parallel tests have run
SELECT COUNT(*) FROM sift_base10k;
SELECT * from sift_base10k WHERE id=4444;
