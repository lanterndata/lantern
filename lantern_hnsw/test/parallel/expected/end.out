-- This file contains invariants to be checked after the parallel tests have run
SELECT COUNT(*) FROM sift_base10k;
 count 
-------
 10030
(1 row)

SELECT * from sift_base10k WHERE id=4444;
  id  |                                                                                                                                                                             v                                                                                                                                                                             
------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 4444 | {55,61,11,4,5,2,13,24,65,49,13,9,23,37,94,38,54,11,14,14,40,31,50,44,53,4,0,0,27,17,8,34,12,10,4,4,22,52,68,53,9,2,0,0,2,116,119,64,119,2,0,0,2,30,119,119,116,5,0,8,47,9,5,60,7,7,10,23,56,50,23,5,28,68,6,18,24,65,50,9,119,75,3,0,1,8,12,85,119,11,4,6,8,9,5,74,25,11,8,20,18,12,2,21,11,90,25,32,33,15,2,9,84,67,8,4,22,31,11,33,119,30,3,6,0,0,0,26}
(1 row)

